pkg/settings/settings.go

```diff
@@ -135,6 +135,12 @@ var (
 		editable,
 		initializeFrom("true"),
 		modifyWith(notBlank))
+	// SkipInsecureVerify indicates whether skip SSL verification when accessing remote server.
+	SkipInsecureVerify = newValue(
```

Comment:
What about change to SkipRemoteTLSVerify

Reply:
updated